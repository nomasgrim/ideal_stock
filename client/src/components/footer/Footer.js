import React from 'react'

import './footer.css'

/**
 * @function Footer
 * @description component for footer
 * @return {ReactElement} JSX
 */
const Footer = () => (
  <div className='footer'>
    <hr />
    <h4>This is the footer</h4>
  </div>
)

export default Footer
