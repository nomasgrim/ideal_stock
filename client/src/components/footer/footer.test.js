/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import Footer from './Footer'

test('Renders the footer component', () => {
  const component = renderer.create(
    <Footer />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
