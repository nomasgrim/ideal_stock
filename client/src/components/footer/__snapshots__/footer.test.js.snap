// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the footer component 1`] = `
<div
  className="footer"
>
  <hr />
  <h4>
    This is the footer
  </h4>
</div>
`;
